tests/acceptance/gcp-auth.bats

```diff
@@ -66,42 +68,42 @@ teardown(){
     } >> $SETUP_TEARDOWN_OUTFILE
 }
 
-@test "Can successfuly write GCP Auth Config" {
-    run vault write auth/gcp/config \
-          credentials="@${PATH_TO_CREDS?}"
+@test "Can successfully write GCP Auth Config" {
+    vault write auth/gcp/config \
```

Comment:
for these commands where we are checking `$status` we will need to use the bats `run` command. Otherwise `$status` won't get set for the command.

Reply:
Whoops, thanks for the catch!